src/crewai/agents/agent_adapters/foundry_agents/foundry_agent_tool_adapter.py

```diff
@@ -0,0 +1,35 @@
+import inspect
+import re
+from typing import Any, List, Optional
+
+from azure.ai.projects.models import FunctionTool
+from crewai.agents.agent_adapters.base_tool_adapter import BaseToolAdapter
+from crewai.tools import BaseTool
+
+class FoundryAgentToolAdapter(BaseToolAdapter):
+    """Adapter for Foundry Assistant tools"""
+
+    def __init__(self, tools: Optional[List[BaseTool]] = None):
+        self.original_tools = tools or []
```

Comment:
I think this method could be removed. You can rely on `BaseToolAdapter` initialization behavior